src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -212,6 +212,11 @@ class ComposeUp extends DefaultTask {
     }
 
     ServiceHost getServiceHost(String serviceName, Map<String, Object> inspection) {
+        String servicesHost = extension.environment['SERVICES_HOST'] ?: System.getenv('SERVICES_HOST')
+        if (servicesHost) {
+            logger.lifecycle("'SERVICES_HOST environment variable detected - will be used as hostname of service $serviceName ($servicesHost)'")
+            new ServiceHost(host: servicesHost, type: ServiceHostType.RemoteDockerHost)
```

Comment:
probably just explictly return this for it work:

```groovy
             return new ServiceHost(host: servicesHost, type: ServiceHostType.RemoteDockerHost)
```

Reply:
also the logging comes out with quotes. You can probably drop the ' and ' within your logger line.

Reply:
Fixed.